api/coreservice.go

```diff
@@ -1765,6 +1790,51 @@ func (core *coreService) Track(ctx context.Context, start time.Time, method stri
 	}, size)
 }
 
+func (core *coreService) traceBlock(ctx context.Context, blk *block.Block, config *tracers.TraceConfig) ([]*apitypes.TxTraceResult, error) {
+	if blk.Height() == 0 {
+		return nil, errors.New("genesis is not traceable")
+	}
+	if err := core.checkActionIndex(); err != nil {
```

Comment:
since the `block` data is provided here, we don't need to check this? i.e., this func should work even if action indexer is not enabled?